### PR TITLE
Network ready scheme

### PR DESCRIPTION
**Pull request by sapience**
Only development scheme is changed at the moment, just to show the progress.

### CODE REVIEW

File: schema.development.kf [hunk 1]
```diff
@@ -314,25 +306,30 @@ action edit_credential($id, $issuer, $credential_type, $credential_level, $crede
 }
 
 action remove_credential($id) public {
-    $has_locked_grants_eth = idos_eth.has_locked_grants(@caller, $id);
-    $has_locked_grants_near = idos_near.has_locked_grants(@caller, $id);
-    $has_locked_grants_arbitrum = idos_arbitrum.has_locked_grants(@caller, $id);
-    $has_locked_grants_etherlink = idos_etherlink.has_locked_grants(@caller, $id);
+    $current_time = idos.current_time();
```
Comment (brennanjl) [inline schema.development.kf:309]: WARNING: This is non-deterministic

Comment (sapience) [reply schema.development.kf]: Thanks, solved.

File: schema.development.kf [hunk 2]
```diff
@@ -219,31 +241,34 @@ action add_wallet($id, $address, $public_key, $message, $signature) public {
     );
 }
 
+// TODO: change to procedure
```
Comment (pkoch) [inline schema.development.kf:244]: What's missing?

File: schema.development.kf [hunk 3]
```diff
@@ -97,8 +80,47 @@ table delegates {
     foreign_key (inserter_id) references inserters(id) on_delete cascade
 }
 
+table access_grants {
+    id uuid primary,
+    ag_owner text notnull,
+    ag_grantee text notnull,
+    data_id text notnull,
+    locked_until int,
```
Comment (pkoch) [inline schema.development.kf:88]: We should make this `notnull`. Maybe we also want other field to not be null, but I know I want this one present for sure.

File: schema.development.kf (hunk 2 above)
Comment (sapience) [reply schema.development.kf:244]: Effort and testing :)

File: schema.development.kf (hunk 3 above)
Comment (sapience) [reply schema.development.kf]: Good point. Done.

File: schema.development.kf (hunk 2 above)
Comment (pkoch) [reply schema.development.kf:244]: Let's punt it it. :+1:

**Review by pkoch — APPROVED** [review]

**Pull request merged**